content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2701,6 +2701,7 @@ Generate a globally unique identifier based on the [`cuid`](https://github.com/e
 
 - Compatible with `String`
 - Implemented by Prisma and therefore not "visible" in the underlying database schema. You can still use `cuid()` when using [introspection](/concepts/components/introspection) by [manually changing your Prisma schema](/concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names) and [generating Prisma Client](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client), in that case the values will be generated by Prisma's [query engine](/concepts/components/prisma-engines/query-engine).
+- Since the length of `cuid()` output is [undefined per the cuid creator](https://github.com/ericelliott/cuid/issues/164), a safe minimum field size is 30 characters, otherwise you may see random Prisma errors like `Error: The provided value for the column is too long for the column's type.`
```

Comment:
should this be "a safe **maximum** field" as the error message talks about "too long"?

Reply:
Ah yeah, totally fine if you want to reword it. I only included the error message because searching for that is how I found the original old closed Github issues that addressed it in the first place. I leave it up to you all though, as your documentation is so well-written already.

Reply:
Proposed rewrite; please review and make sure it is still technically accurate:
"Since the length of `cuid()` output is undefined per the cuid creator, a safe minimum field size is 30 characters, in order to allow for enough characters for very large values. If you set the minimum field size as less than 30, and then a large value is entered, you might see Prisma errors such as `Error: The provided value for the column is too long for the column's type.`

Reply:
Looks just about right. I'd reword just this one part myself:

"...If you set the minimum field size as less than 30, and then a larger value is generated by `cuid()`, you might see..."

(as when using `cuid()` in Prisma, we are never entering the values ourselves, they're auto-generated)

Reply:
Thanks so much for that important clarification, @amcjen. Do you want to make/commit that change please, then we can go ahead and merge this PR?

Reply:
Updated!